src/towncrier/_shell.py

```diff
@@ -9,17 +9,24 @@
 
 from __future__ import annotations
 
+from importlib.metadata import PackageNotFoundError, version
+
 import click
 
-from ._version import __version__
```

Comment:
Why not keep the import ?

It's just that now `__version__` will be a simple text.

Reply:
done